### PR TITLE
Support PHP 8.1.0

### PR DESCRIPTION
**Pull request by asika32764**
Add `#[\ReturnTypeWillChange]` to avoid type error.

### CODE REVIEW

**Comment by Ocramius** [comment]
Need 8.1 in the pipeline to validate this fix

**Comment by Ocramius** [comment]
Hmm, seems like this will wait for upstream support for `8.1` in builds (probably happening in September, after RC builds are available - ping me then :) )

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
+          - "8.1"
```
Comment (cedric-anne) [inline .github/workflows/phpunit.yml]: I guess alpine `8.1` tag will not be available without a `-rc` suffix before the official release of PHP 8.1 stable version. So if you want to run the build now, you have to add this suffix.
```suggestion
          - "8.1-rc"
```

Comment (Nek-) [reply .github/workflows/phpunit.yml]: Maybe "8.1.0RC3" is even more accurate. (not sure about it)

Comment (cedric-anne) [reply .github/workflows/phpunit.yml]: Ping @Ocramius

**Comment by Ocramius** [comment]
LGTM :+1: 

We can upgrade to `8.0` post-release, if the pipeline decides to fail for whatever reason :-)

**Review by Ocramius — APPROVED** [review]
:ship:

**Pull request merged**